.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+*                                                                                 @dotnet/dotnet-upgrade-assistant-admin
```

Comment:
Super nit-pick, there seems to a be a lot of unnecessary white space on this line after the `*`

Reply:
reduced some of the spaces :)